ivy_tests/test_ivy/test_frontends/test_jax/test_devicearray.py

```diff
@@ -1088,7 +1088,7 @@ def test_jax_devicearray__rxor_(
         available_dtypes=st.one_of(st.just(("bool",)), helpers.get_dtypes("integer")),
     ),
 )
-def test_jax_devicearray__invert_(
+def test_jax__invert_(
```

Comment:
I think you have missed some underscores in the special methods tests. For example, here the name of the function is `__invert__` so the test should be `test_jax___invert__`.